libs/pausable/README.md

```diff
@@ -0,0 +1,103 @@
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset=".docs/pausable-logo-dark-theme.png">
+        <img alt="pausable_logo_light" width="400px" src=".docs/pausable-logo-light-theme.png">
+    </picture>
+</p>
+
+# Overview
+
+The Pausable library allows contracts to implement an emergency stop mechanism. This can be useful for scenarios such as having an emergency switch to freeze all transactions in the event of a large bug.
+
+> **NOTE** It is highly encouraged to use the [Ownership Library](../ownership/) in combination with the Pausable Library to ensure that only a single administrative user has the ability to pause your contract.
+
+More information can be found in the [specification](./SPECIFICATION.md).
+
+# Using the Library
+
+## Getting Started
+
+In order to use the Pausable library it must be added to the `Forc.toml` file and then imported into your Sway project. To add Pausable as a dependency to the `Forc.toml` file in your project please see the [README.md](../../README.md).
+
+You may import the Pausable library's functionalities like so:
+
+```sway
+use pausable::*;
+```
+
+Be sure to add the storage block bellow to your contract which enables the library.
```

Comment:
```suggestion
Be sure to add the storage block below to your contract which enables the library.
```

How about we write directly to some storage slot using read/write? No need for them to use a storage field then

Reply:
We used to do this before we had `StorageKey`s. Might not be the worst thing to implement again.